source/extensions/filters/network/dubbo_proxy/hessian_utils.cc

```diff
@@ -516,6 +564,16 @@ std::string HessianUtils::readByte(Buffer::Instance& buffer) {
   return result;
 }
 
+size_t HessianUtils::writeString(Buffer::Instance& buffer, const absl::string_view str) {
```

Comment:
ditto

Reply:
I have modified it to: size_t writeString(Buffer::Instance& buffer, absl::string_view str).